app/features/router/history.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+import { createHashHistory } from 'history';
```

Comment:
why do we need this hash history thing if we are not using (or are we?) a web based router? I thought the redux router may allow us to not use hash based routing.

Reply:
`react-router-redux` is still web based. This is just somehow linking history object (which contains all the information about browser session) to redux store using middleware and reducer.

You can see usage here.
https://github.com/ReactTraining/react-router/tree/master/packages/react-router-redux#usage

Reply:
👍